### PR TITLE
Add a disclosure dialog

### PR DESCRIPTION
**Pull request by Montymakes**
Adds a dialog that explains that the summaries for each entry are generated by AI and that nothing on the page should be considered legal advice and some general advice on how to best use the app. Users must accept before accessing the site. 

Also, deleted some unnecessary files from gitignore.

### CODE REVIEW

File: src/components/Header/DisclosureDialog.js
Comment (kylebeloin) [inline src/components/Header/DisclosureDialog.js:8]: Small suggestion, and really a preference around being more concise (just referencing the package folder)  vs. more explicit (referencing the package/component folder) . You could import these like this:
```
import {
  Button,
  Dialog,
  DialogActions,
  DialogContent,
  DialogContentText,
  DialogTitle,
} from '@mui/material'
```

**Review by kylebeloin — COMMENTED** [review]
This looks so good Monty! I especially like how the disclosure is accessible via the nav bar whenever someone wants to open it again.

One thing I noticed is that the dialog is dismissible when clicking on the backdrop. I wonder if it makes sense to force a user to click on the "Accept" button to dismiss? What do you think?

I'd approve this if I could! But I don't have approver privileges...

**Review by mike-feeney — APPROVED** [review]
I agree with Kyle this looks great! Thank you! I'm approving but I'll wait to merge in case you want to incorporate Kyle's feedback.

**Comment by mike-feeney** [comment]
I'll merge this now.

**Pull request merged**
